backend/src/main/java/heartbeat/service/report/CSVFileGenerator.java

```diff
@@ -102,7 +103,7 @@ public void convertPipelineDataToCSV(List<PipelineCSVInfo> leadTimeData, String
 		createCsvDirToConvertData();
 
 		String fileName = CSVFileNameEnum.PIPELINE.getValue() + FILENAME_SEPARATOR + csvTimeStamp + CSV_EXTENSION;
-		if (!fileName.contains("..") && fileName.startsWith(FILE_LOCAL_PATH)) {
+		if (!fileName.contains("..")) {
```

Comment:
duplicated code
`fileName.contains("..")`

Reply:
I think it's not necessary to delete the code because if the `csvTimeStamp` contains `..`, there will throw exception, although this is impossible.